Source/Events.Processing/Filters/ValidateFilterForAllTenants.cs

```diff
@@ -50,21 +45,10 @@ public class ValidateFilterForAllTenants : IValidateFilterForAllTenants
             await _onAllTenants.PerformAsync(async tenantId =>
                 {
                     var filterProcessor = getFilterProcessor();
-                    _logger.TryGetFilterDefinition(filterProcessor.Identifier, tenantId);
-                    var tryGetFilterDefinition = await _getFilterDefinitions().TryGetFromStream(filterProcessor.Scope, filterProcessor.Definition.TargetStream, cancellationToken).ConfigureAwait(false);
-                    if (tryGetFilterDefinition.Success)
-                    {
-                        _logger.ValidatingFilterForTenant(filterProcessor.Identifier, tenantId);
-                        var validationResult = await _filterValidators.Validate(tryGetFilterDefinition.Result, filterProcessor, cancellationToken).ConfigureAwait(false);
-                        result.Add(tenantId, validationResult);
-                    }
-                    else
-                    {
-                        _logger.NoPersistedFilterDefinition(filterProcessor.Identifier, tenantId);
-                        result.Add(tenantId, new FilterValidationResult());
-                    }
+                    _logger.ValidatingFilterForTenant(filterProcessor.Identifier, tenantId);
+                    var validationResult = await _filterValidators.Validate(filterProcessor, cancellationToken).ConfigureAwait(false);
+                    result.Add(tenantId, validationResult);
                 }).ConfigureAwait(false);
```

Comment:
This can probably be done in "parallel"? I think that we have a method on _onAllTenants that does a Task.WhenAll(tasks) underneath. Maybe we could consider using it here?